lib/api.js

```diff
@@ -222,6 +232,7 @@ app.set('port', process.env.PORT || 3000);
 
 server.listen(app.get('port'), function() {
   logger.info("✔ Aggie is listening on port " + app.get('port'));
+  logger.info("✔ Aggie is listening to protocol " + protocol);
```

Comment:
Is there a reason to remove the port number? Might be useful for those on http. Perhaps just add the protocol in addition?